docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst

```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
        $ aws s3api get-bucket-lifecycle-configuration --bucket zenko-bucket
 
 Once the lifecycle rules on the bucket are set, the rules apply to all
-objects in the specified bucket.
\ No newline at end of file
+objects in the specified bucket.
+
+.. _Lifecycle Queries:
+
+Querying Lifecycle Events
+-------------------------
+
+A lifecycle metrics feature is not available for release 1.1. The only
+available lifecycle event metric is to send a head-object request to
```

Comment:
I'm not sure I'd go so far as to characterize the head object response as a "metric". Maybe we can say something like "The storage location of the transitioned object data can be accessed by viewing the object's metadata, for example by making a head object request".

Reply:
done